atlas-starter.py

```diff
@@ -0,0 +1,91 @@
+import pymongo
+
+# Replace the placeholder data with your Atlas connection string. Be sure it includes
+# a valid username and password! Note that in a production environment,
+# you should not store your password in plain-text here.
+
+client = pymongo.MongoClient(
+      "mongodb+srv://<username>:<password>@<cluster-name>/test?retryWrites=true&w=majority")
+
+# use a database named "my_database"
+
+db = client.my_database
+
+# use a collection named "recipes"
+
+my_collection = db["recipes"]
+
+recipe_documents = [{ "name": "elotes", "ingredients": ["corn", "mayonnaise", "cotija cheese", "sour cream", "lime"], "prep_time": 35 },
+                    { "name": "loco moco", "ingredients": ["ground beef", "butter", "onion", "egg", "bread bun", "mushrooms"], "prep_time": 54 },
+                    { "name": "patatas bravas", "ingredients": ["potato", "tomato", "olive oil", "onion", "garlic", "paprika"], "prep_time": 80 },
+                    { "name": "fried rice", "ingredients": ["rice", "soy sauce", "egg", "onion", "pea", "carrot", "sesame oil"], "prep_time": 40 }]
+
+# drop the collection in case it already exists
+
+my_collection.drop()  
```

Comment:
I don't have a strong opinion on this, but our other examples don't do this and it is a dangerous potential footgun for new users. I could take it or leave it.

Reply:
It makes the app idempotent. Without it, the records will pile up in the 'recipes' collection and you'll get strange-looking results if you run the app more than once. Are the other apps really just adding more and more records to the collection?

Reply:
It's certainly behaviour we can change. Right now, I believe that's what java/python/dotnet all do.